src/e2e-test/resources/pluginDataCyAttributes.properties

```diff
@@ -0,0 +1,27 @@
+## DATA-CY ATTRIBUTES-DATAPLEX-PLUGIN-START
+dataplexReferenceNameID=referenceName

```

Comment:
dataplexReferenceNameID -> dataplexReferenceName

It should be as it gets displayed on UI. 
Check all properties..u have added suffix `ID` for all.